lib/rules/no-array-prototype-extensions.js

```diff
@@ -278,6 +268,42 @@ function applyFix(callExpressionNode, fixer, context, options = {}) {
 
       return [];
     }
+    case 'without': {
+      // Get the open parenthesis immediately after the callee name
+      const openParenToken = sourceCode.getTokenAfter(calleeProp, {
+        filter(token) {
+          return token.type === TOKEN_TYPES.PUNCTUATOR && token.value === '(';
+        },
+      });
+      // Get the close parenthesis from the end of the callExpressionNode
+      const closeParenToken = sourceCode.getLastToken(callExpressionNode, {
+        filter(token) {
+          return token.type === TOKEN_TYPES.PUNCTUATOR && token.value === ')';
+        },
+      });
+
+      if (callArgs.length === 1) {
+        const argText = sourceCode.getText(callArgs[0]);
+        const calleeObjText = sourceCode.getText(calleeObj);
+
+        return [
+          // As per https://api.emberjs.com/ember/release/classes/EmberArray/methods/mapBy?anchor=without
+          // when the passed value doesn't not exist in the array, it returns the original array
+          // Hence we first check for the existence of the passed value in the array before calling filter on array
+          // Used indexOf instead of includes since v3.x of ember is committed to supporting IE11
+          // as per the ember browser support policy (https://emberjs.com/browser-support/)
```

Comment:
```suggestion
          // Used indexOf instead of includes since v3.x of ember is committed to supporting IE11
          // as per the ember browser support policy (https://emberjs.com/browser-support/)
          // TODO: Switch to includes once Ember v3 LTS support ends: https://emberjs.com/releases/lts/
```

While we're waiting for Ember v3 LTS support to end, the good news is that [unicorn/prefer-includes](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-includes.md) will autofix this for us.

Reply:
Added the TODO.